tutorindigo/templates/indigo/lms/templates/footer.html

```diff
@@ -102,21 +98,16 @@
         ## through an API to partner sites (such as marketing sites or blogs),
         ## which are not technically powered by OpenEdX.
         % if not hide_openedx_link:
-        <ul class="logo-list">
+        <ul class="footer-column logo-list">
           <li>
             <a href="https://www.uah.es/es/" rel="noopener" target="_blank">
               <img src="${static.url('images/tutor-logo.png')}" alt="img" width="140" />
```

Comment:
In the figma I saw a second image along side this one (aniversario patrimonio mundial) that image is not required for the present changes?

Reply:
Thanks for noticing! No, it is not necessary.

Extract of feedback from client (more info in the PR description):

> En cuanto al diseño, está bien tal cual, solo habría que quitar el logo de 25 años de abajo (porque ya no se cumple), pero por lo demás, todo ok 🙂